src/app/api/organizations/[organizationId]/route.ts

```diff
@@ -24,3 +27,43 @@ export async function DELETE(
   }
   return new NextResponse(undefined, { status: 204 });
 }
+
+// @route PUT /api/organizations/[organizationId] - updates an existing organization in the database
+export async function PUT(
+  _request: NextRequest,
+  { params }: { params: { organizationId: string } }
+) {
+  try {
+    const validationResult = zObjectId.safeParse(params.organizationId);
+    if (!validationResult.success) {
+      throw new Error('Invalid Organization Id');
```

Comment:
Instead of throwing in the API we can just directly call `return NextResponse.json({ message: "Invalid Organization Id" }, { status: 400 })`